src/core/env.js

```diff
@@ -121,7 +121,7 @@ const defaults = new Map(
     // secondary doh upstream
     CF_DNS_RESOLVER_URL_2: {
       type: "string",
-      default: "https://dns.google/dns-query",
+      default: "https://dns.nextdns.io/dns-query",
```

Comment:
perhaps meant for your private repo?

Reply:
> perhaps meant for your private repo?

Oh no🤣. I didn’t realize it brought all my commits over here. So sorry about that. Il make a new pull request requesting to update the original stuff I planned to have here when deno 1.31.0 comes out next week. Again, sorry